docs/github-pages-example-2.html

```diff
@@ -1,15 +1,17 @@
 <div>
-    <div id="wrapper-1">
+    <div id="wrapper-2">
     </div>
 
     <script type="text/javascript">
         dtCreateDataTable({
-            wrapperDivId: 'wrapper-1',
-            numRows: 6,
-            numColumns: 6,
+            wrapperDivId: 'wrapper-2',
+            numRows: 4,
+            numColumns: 3,
             rowsNamePlural: 'fruits',
-            columnsName: 'Round',
-            rowsName: 'Type of fruit'
+            columnsName: 'Day',
+            rowsName: 'Type of fruit',
+            names: ["# Value", "Status"],
```

Comment:
What does `# value` mean in reference to fruit? Does it mean `# in stock` or `price`?

(FWIW I think price is better, since there's already an in-stock/out-of-stock dropdown)